troposphere/awslambda.py

```diff
@@ -12,7 +18,7 @@
     'AWS_LAMBDA_FUNCTION_NAME',
     'AWS_LAMBDA_FUNCTION_VERSION',
     'AWS_LAMBDA_LOG_GROUP_NAME',
-    'AWS_LAMBDA_LOG_STREAM_NAME',
+    'AWS_LAMBDA_LOGbasestringEAM_NAME',
```

Comment:
assuming this was accidental?
```suggestion
    'AWS_LAMBDA_LOG_STREAM_NAME',
```

Reply:
Uh..lemme fix that!